packages/sanity/src/core/comments/components/list/CommentsListItem.tsx

```diff
@@ -186,8 +186,7 @@ export function CommentsListItem(props: CommentsListItemProps) {
             onChange={setValue}
             onEditDiscard={cancelEdit}
             onSubmit={handleSubmit}
-            placeholder="Reply…"
-            ref={replyInputRef}
```

Comment:
We should keep this ref so that `replyInputRef.current?.focus()` still works in `handleSubmit`.